src/pages/NotFoundPage.tsx

```diff
@@ -0,0 +1,30 @@
+import { Link } from "react-router-dom";
+import NotFoundButton from "../components/NotFoundButton";
+import notFoundImage from "/images/404.png";
+
+export default function NotFoundPage() {
+  return (
+    <div className="flex flex-col items-center justify-center min-h-screen bg-[var(--color-background)] text-[var(--color-primary)] text-center px-6">
+      
+      <h1 className="text-[40px] font-bold leading-[100%] max-w-[280px] mt-[135px]">
```

Comment:
I would look into if it's possible to use native Tailwind here without providing exact values.